AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -78,10 +78,21 @@ - (BOOL)dropTable:(NSString *)tableName {
 }
 
 - (BOOL)createTable:(NSString *)tableName columnsSchema:(MSDBColumnsSchema *)columnsSchema {
+  return [self createTable:tableName columnsSchema:columnsSchema uniqueColumnsConstraint:nil];
+}
+
+- (BOOL)createTable:(NSString *)tableName
```

Comment:
run formatter, this doesn't look right.  `:`s should align

Reply:
I re-ran the formatter but this didn't change. I assume it's because the first line is too short for the following lines' `:` to line up with it.

Reply:
Aligning parameters in Objective-C is really weird. 😆